README.md

```diff
@@ -22,13 +23,14 @@ To set up the branch:
 
 To build the storm GIF:
 1. Make your best guess about an appropriate `bbox` in viz_config.yml.
-1. Make your best guess about a good site list in 2_process/src/filter_sites_custom.R (this can be broad, at the cost of needing to pull and plot more NWIS data in early iterations).
+1. Make your best guess about a good site list in 2_process/src/filter_sites_custom.R (this can be broad, at the cost of needing to pull and plot more NWIS data in early iterations). It can be helpful to see the site numbers on the figures with the spark lines as you iterate through site selection. One way to do this is to add this `text(full_poly$x[1]-strwidth("5555"), full_poly$y[1], labels=site)` to the `prep_spark_line_fun` where the stage shapes are being added to the plot. Just delete when you're done iterating. 
 1. Build 6_storm_gif_tasks.yml with `scmake('6_storm_gif_tasks.yml')`.
 1. Choose a GIF frame to build to preview the vizzy setup. The dates and therefore exact filenames will differ, but you may want to build one gif_frame and one gif_TEST_frame (which shows the bbox) to start. For example, you might run `scmake('6_visualize/tmp/gif_frame_a_20170825_00.png', '6_storm_gif_tasks.yml')` and/or `scmake('6_visualize/tmp/gif_TEST_frame_a_20170825_00.png', '6_storm_gif_tasks.yml')`
 1. If jumping directly to building a gif_frame, you may find that some steps (especially fetches from the internet) require retries. It may boost your overall build efficiency to `scmake` some of these steps individually rather than relying on the full `scipiper` chain, mainly because of known issues https://github.com/USGS-R/scipiper/issues/70 and https://github.com/USGS-R/scipiper/issues/47.
 1. When git committing your changes, make sure that any `*.ind` file you commit (new or modified) has a matching `build/status/*.yml` file in your commit. If it doesn't, run `scipiper:::YAMLify_build_status('yourindfile.ind')`. The need to do this reflects the scipiper issues noted just above.
 1. Iterate through choices of bbox, site list, ocean and city names, etc. until you have the gif_frames (pngs) looking the way you want.
 1. Build the animated GIF with `scmake('6_visualize/out/animation_a.gif')`. It often takes a long time to build all the pngs, so if you want to preview an animated GIF with only some frames, consider specifying a subset of frames with the `task_names` argument to `combine_animation_frames()`.
+1. Once you're happy with the way the animated GIF looks, you will need to build the intro frames. Do this by making sure `task_names=NULL` for the target `6_visualize/log/6_intro_gif_tasks.ind` in `6_visualize.yml`. Run `scmake('6_visualize/log/6_intro_gif_tasks.ind')` to build the intro frames. Then, run `scmake(6_visualize/out/animation_a.gif')` to incorporate the intro frames into the full animated GIF (note that you might need to add `force = TRUE` if the file already exists).
```

Comment:
maybe add that you might need force=TRUE not just if animation_a.gif already exists but also if '6_visualize/log/6_intro_gif_tasks.ind' or '6_visualize/log/6_storm_gif_tasks.ind' already exist but the pngs still need [re]building?

Reply:
oh right, good point